code/Player/Controllers/HumanController.cs

```diff
@@ -83,17 +89,34 @@ protected virtual void TryPlayerMove()
 				Velocity = Vector3.Zero;
 				return;
 			}
-			
-			MoveHelper mover = new(Position, Velocity);
-			mover.Trace = mover.Trace.Size(mins, maxs).Ignore(Pawn);
-			mover.MaxStandableAngle = GroundAngle;
+
+			MoveHelper mover = GetMoveHelper();
 
 			mover.TryMove(Time.Delta);
 			
 			Position = mover.Position;
 			Velocity = mover.Velocity;
 		}
+		
+		/// <summary>
+		/// Moves the player in the direction of their velocity and if there is colliding objects, checks if it can step.
+		/// </summary>
+		protected virtual void TryPlayerMoveWithStep()
+		{
+			if (Velocity.Length <= 1.0F)
```

Comment:
magic number

Reply:
it's for ignoring small movements, it's nauwp's idea

Reply:
Yeah i know. Magic number means: all constant have to be declared outside the scope. In this example you have to declare and init this value in the properties of the class as const